randovania/games/am2r/exporter/patch_data_factory.py

```diff
@@ -413,28 +414,35 @@ def _credits_spoiler(self) -> str:
 
         return spoiler
 
+    def create_memo_data(self) -> dict:
+        """Used to generate pickup collection messages."""
+        text_data = self._get_text_data()
+        memo_data = {key: value["text_desc"] for key, value in text_data.items()}
+        memo_data["Energy Tank"] = memo_data["Energy Tank"].format(Energy=self.configuration.energy_per_tank)
+        return memo_data
+
+    def create_useless_pickup(self) -> PickupEntry:
+        """Used for any location with no PickupEntry assigned to it."""
+        return pickup_creator.create_nothing_pickup(
+            self.game.resource_database,
+            model_name="sItemNothing",
+        )
+
+    def create_visual_nothing(self) -> PickupEntry:
+        """The model of this pickup replaces the model of all pickups when PickupModelDataSource is ETM"""
+        return pickup_creator.create_visual_nothing(self.game_enum(), "sItemUnknown")
+
     def create_game_specific_data(self) -> dict:
         db = self.game
 
-        useless_target = PickupTarget(
+        PickupTarget(
```

Comment:
leftover? No variable assigned to it and seems to be done by `create_useless_pickup`

Reply:
ooops